src/main/kotlin/it/pagopa/wallet/cdc/PaymentWalletsLogEventsStream.kt

```diff
@@ -75,7 +76,7 @@ class PaymentWalletsLogEventsStream(
                             retryStreamPolicyConfig.maxAttempts,
                             Duration.ofMillis(retryStreamPolicyConfig.intervalInMs)
                         )
-                        .filter { t -> t is MongoServerException }
+                        .filter { t -> t is MongoServerException || t is MongoSocketException }
```

Comment:
both extends `MongoException`: what about perform retry for all MongoException?

Reply:
Done in [ebd842d](https://github.com/pagopa/pagopa-payment-wallet-cdc-service/pull/18/commits/ebd842d7740acee8b25727d6293aa4589254a51d)